src/Settings/Settings/SettingsGeneral.php

```diff
@@ -150,7 +150,8 @@ public function getSettingsGlobalData(): array
 					Select options you want to disable on your forms. <br /><br />
 					Keep in mind that you know what you are doing and that you will provide your own styles and scripts.<br /><br />
 					Disabling styles will remove all of plugin styles for frontend and block editor.<br /><br />
-					Disabling scripts will remove all scripts on the frontend, keep in mind this will make the frontend validation and form submitting stop working.
+					Disabling scripts will remove all scripts on the frontend, keep in mind this will make the frontend validation and form submitting stop working. <br /><br />
```

Comment:
```suggestion
					Disabling scripts will remove all scripts on the front end. Keep in mind this will make the frontend validation and submitting the form stop working. <br /><br />
```